stripe/src/main/java/com/stripe/android/model/StripeIntent.kt

```diff
@@ -50,6 +50,16 @@ interface StripeIntent : StripeModel {
 
     val nextActionData: NextActionData?
 
+    /**
+     * Confirmation has succeeded and all required actions have been handled.
+     */
+    val isConfirmed: Boolean
+        get() = setOf(
+            Status.Processing,
+            Status.RequiresCapture,
+            Status.Succeeded
+        ).contains(status)
```

Comment:
Intentionally public?

Reply:
They could be internal, but we'll have to make them public anyway if we move payment sheet to a separate module.

Reply:
It might be worth waiting to make them public until that time because then you won't have people inadvertently using them and need to migrate.